src/test/regress/expected/multi_partitioning.out

```diff
@@ -641,7 +647,7 @@ SELECT name, type FROM table_attrs WHERE relid = 'partitioning_test_2010'::regcl
 -- test add PRIMARY KEY
 -- add PRIMARY KEY to partitioned table - this will error out
 ALTER TABLE partitioning_test ADD CONSTRAINT partitioning_primary PRIMARY KEY (id);
-ERROR:  insufficient columns in PRIMARY KEY constraint definition
+ERROR:  unique constraint on partitioned table must include all partitioning columns
```

Comment:
can we use something like below to avoid additional files?
```
CREATE OR REPLACE FUNCTION raise_failed_execution(query text) RETURNS void AS $$
BEGIN
	EXECUTE query;
	EXCEPTION WHEN OTHERS THEN
	IF SQLERRM LIKE 'failed to execute task%' THEN
		RAISE 'Task failed to execute';
	END IF;
END;
$$LANGUAGE plpgsql;
```
(see regression tests for the examples)

Reply:
What about a normalization rule like:
s/unique constraint on partitioned table must include all partitioning columns/insufficient columns in PRIMARY KEY constraint definition/g

With this rule we wouldn't need to change any test file

Reply:
I verified that the former error message is no longer used in >= PG13.1 so this substitution is ok to do.

Reply:
It seems that the only failure is indeed this: https://app.circleci.com/pipelines/github/citusdata/citus/15309/workflows/9d625a29-a138-4207-bceb-3ac463b22b79

@hanefi can you cherry-pick https://github.com/citusdata/citus/commit/ef9a477d343937b68495243ebb72163cf3966cef on top of this PR and see if tests pass? They should pass, then we can update the images as a separate commit in this PR

Reply:
I made a change on your suggested normalization rule:

instead of `s/new/old/` I used `s/^old$/new/` that made more sense to me.

I also cherry-picked the commit to run the tests on PG13.2

Reply:
>instead of s/new/old/ I used s/^old$/new/ that made more sense to me.

Makes sense. Can you remove the `-dev202104301224` suffix from the images and then fix the commit message `wip`. I think then we can merge the PR. We will push the production images

Reply:
@onderkalaci is the approach we chose okay for you as an alternative to your suggestion?

Reply:
In general, I prefer to avoid additional lines on the normalization, as it also adds some overhead to the test execution times. And, we never get to remove these lines.

So, I have a slight preference to change the test, but this is also fine.